support-frontend/assets/helpers/abTests/abtest.ts

```diff
@@ -72,11 +71,44 @@ export type Test = {
 
 export type Tests = Record<string, Test>;
 
-// ----- Setup ----- //
+// ----- Init ----- //
+
+export function init(
+	country: IsoCountry,
+	countryGroupId: CountryGroupId,
+	settings: Settings,
+	abTests: Tests = tests,
+	mvt: number = getMvtId(),
+	acquisitionDataTests: AcquisitionABTest[] = getTestFromAcquisitionData() ??
+		[],
+): Participations {
```

Comment:
I've moved this function to the top of the file as it's the most important one. I've also added a couple extra optional arguments (`mvt` and `acquisitionDataTests`) to make it easier to test